src/main/java/org/usfirst/frc/team2473/robot/Robot.java

```diff
@@ -42,12 +46,12 @@ public void robotInit() {
 		// cvLight = new Relay(RobotMap.CV_LIGHT);
 		prefs = Preferences.getInstance();
 				
-		UsbCamera cubeCam = CameraServer.getInstance().startAutomaticCapture("Cube View", 0);
-		cubeCam.setBrightness(75);
-		cubeCam.setResolution(640, 480);
-		UsbCamera driveCam = CameraServer.getInstance().startAutomaticCapture("Front View", 1);
-		driveCam.setBrightness(75);
-		driveCam.setResolution(640, 480);
+		// UsbCamera cubeCam = CameraServer.getInstance().startAutomaticCapture("Cube View", 0);
```

Comment:
This appears to be unrelated to this change

Reply:
Perhaps it was for testing?